.travis.yml

```diff
@@ -28,11 +24,6 @@ env:
 
 sudo: false
 
-before_cache:
```

Comment:
Why remove the `before_cache` section?  `modules-2.lock` changes on each build, AFAIK.